src/transformStateWithClones.js

```diff
@@ -7,7 +7,28 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const transformedState = structuredClone(state);
```

Comment:
The use of `structuredClone` is not necessary and is not the most efficient way to copy objects in this context. It's better to use the spread operator or `Object.assign` for shallow cloning.

Reply:
I can't pass tests without structuredClone